docs/source/hyperparameters.rst

```diff
@@ -105,21 +105,19 @@ modify the network and read those values in the LightningModule
 
     class LitMNIST(LightningModule):
 
-        def __init__(self, hparams):
+        def __init__(self, layer_1_dim, layer_2_dim, learning_rate, batch_size):
             super().__init__()
+            self._auto_reguster_arguments()
```

Comment:
we are NOT doing this anymore. 

users need to explicitely assign variables
ie:
```python
# THIS WILL FAIL
self.layer_1_dim
```
Unless the user does:
```python
self.layer_1_dim = layer_1_dim
```

Reply:
this has to be called explicitly in the model, just lazy to write all by hand...

Reply:
the PR description is outdated